RDMMonitor.js

```diff
@@ -558,11 +551,143 @@ function SendOfflineDeviceDMs()
 
     setTimeout(SendOfflineDeviceDMs,60000);
     return;
-    
+}
+
+function ReopenWarnGame()
+{
+    if(!config.allowReopenGame)
+    {
+        return;
+    }
+    else
+    {

```

Comment:
```suggestion
    if(!config.allowReopenGame)
    {
        return;
    }
```
Remove unnecessary `else` statement